### ISSUE TITLE
Latest version 4.5.4.4788 not working with See3CAM_CU135 - 4K USB Camera Board

### ISSUE THREAD

**Issue by sysmith1** [emgucv/emgucv#681]
I recently downloaded the latest release of the Emgu CV 4.5.4 and then loaded Emgu.CV.Example.sln solution and built all the projects.  I selected the CameraCapture project as the startup project and ran the project.  The camera's activation led did not light when connected to the camera and no ProcessFrame events occured. I then unplugged all attached cameras except for my WebCam and then ran the CameraCapture application again.  This time it behaved as expected.  I then unplugged the WebCam and plugged in one of the cameras I'm using for my project, an e-con Systems See3CAM_CU135 - 4K USB Camera Board and ran the CameraCapture application again.  This time no images are being captured.  The _capture.IsOpened property was set to true after the call to            _capture = new VideoCapture(); but the ProcessFrame event was never called.  The e-con System cameras work fine when run in their native application.            

After upgrading my application to the latest release of Emgu CV, my software now cannot access the e-con system cameras.
I had no problems accessing the e-con System cameras when I used the old version Emgu.CV.4.2.0.3662.

Any suggestions on how to resolve this issue?

Using emgucv-windesktop 4.5.4.4788 on Windows 10, .Net Framework 4.6.1, x86_64 (AnyCPU)

To Reproduce: attach Attach an e-con Systems camera and build and run the CameraCapture example provided in emgu.cv download.
Does it capture images?

Expected behavior: The CameraCapture application should be able to display a live stream of images from camera.

**Comment by emgucv** [comment]
Once the VideoCapture is created, what is the "BackendName" in 4.2.0?
https://www.emgu.com/wiki/files/4.2.0/document/html/34b665d9-38f9-ef88-9e7f-58d3830281ab.htm

and what is the "BackendName" in 4.5.4?
https://www.emgu.com/wiki/files/4.5.4/document/html/P_Emgu_CV_VideoCapture_BackendName.htm
 
If the backend in 4.5.4 is different from 4.2.0, try to create the VideoCapture in 4.5.4 using the backend type from 4.2.0

**Comment by sysmith1** [comment]
The BackendName is the same in 4.2.0 as 4.5.5 ("MSMF"). 
![image](https://user-images.githubusercontent.com/51513423/148586975-ba081e77-1e69-44f0-9eab-3b178569fe77.png)
Any other suggestions?

**Comment by sysmith1** [comment]
More info: I tested all versions from Emgu CV 4.5.4 back to Emgu CV 4.2.0 and found 4.2.0 to be the only release that captures and display images from the See3CAM_CU135 camera.  I did not test prior to 4.2.0.

Is it necessary to bump up the .net framework above 4.6.1 for this to work?  I wouldn't think so.
Is the emgu.cv software basically a managed wrapper around OpenCV?
Is there a way for me to test if the error in the underlying OpenCV code or in the emgu.cv code?  If so, how?

The reason that prompted me to attempt an upgrade was that I was having a problem with calling the Image.Copy(Rectangle roi) method.  The issue happens when I copy only one row of pixels. The pixel values are not correct. I know the sequence of pixels values in the origingal image but the copied pixel values are all 255 except for the last 3 pixels copied.    If I copy two or more rows, the pixel values are correct.

My preference would be to upgrade to the latest and be able to capture and display images and be able to do a copy and all the correct pixels values get copied.  Is there any tests I can run to help figure where the problem lies.
